readme.md

```diff
@@ -319,28 +319,31 @@ This is `undefined` if either:
 - the [`all` option](#all-2) is `false` (the default value)
 - both [`stdout`](#stdout-1) and [`stderr`](#stderr-1) options are set to [`'inherit'`, `'ipc'`, `'ignore'`, `Stream` or `integer`](https://nodejs.org/api/child_process.html#child_process_options_stdio)
 
-#### pipeStdout(target)
+#### pipeStdout(execaChildProcess)
 
-[Pipe](https://nodejs.org/api/stream.html#readablepipedestination-options) the child process's `stdout` to `target`, which can be:
-- Another [`execa()` return value](#pipe-multiple-processes)
-- A [writable stream](#save-and-pipe-output-from-a-child-process)
-- A [file path string](#redirect-output-to-a-file)
+`execaChildProcess`: [`execa()` return value](#pipe-multiple-processes)
 
-If the `target` is another [`execa()` return value](#execacommandcommand-options), it is returned. Otherwise, the original `execa()` return value is returned. This allows chaining `pipeStdout()` then `await`ing the [final result](#childprocessresult).
+[Pipe](https://nodejs.org/api/stream.html#readablepipedestination-options) the child process' `stdout` to another Execa child process' `stdin`.
 
-The [`stdout` option](#stdout-1) must be kept as `pipe`, its default value.
+Returns `execaChildProcess`, which allows chaining `pipeStdout()` then `await`ing the [final result](#childprocessresult).
 
-#### pipeStderr(target)
+[`childProcess.stdout`](#stdout) must not be `undefined`.
 
-Like [`pipeStdout()`](#pipestdouttarget) but piping the child process's `stderr` instead.
+#### pipeStderr(execaChildProcess)
 
-The [`stderr` option](#stderr-1) must be kept as `pipe`, its default value.
+`execaChildProcess`: [`execa()` return value](#pipe-multiple-processes)
 
-#### pipeAll(target)
+Like [`pipeStdout()`](#pipestdoutexecachildprocess) but piping the child process's `stderr` instead.
 
-Combines both [`pipeStdout()`](#pipestdouttarget) and [`pipeStderr()`](#pipestderrtarget).
+[`childProcess.stderr`](#stderr) must not be `undefined`.
 
-Either the [`stdout` option](#stdout-1) or the [`stderr` option](#stderr-1) must be kept as `pipe`, their default value. Also, the [`all` option](#all-2) must be set to `true`.
```

Comment:
The first sentence is redundant, since we already mention that this method combines both `pipeStdout()` and `pipeStderr()` which already document that requirement.